vgscollect/src/main/java/com/verygoodsecurity/vgscollect/core/VGSCollect.kt

```diff
@@ -707,4 +702,8 @@ class VGSCollect {
     fun setAnalyticsEnabled(isEnabled: Boolean) {
         tracker.setAnalyticsEnabled(isEnabled)
     }
+
+    companion object {
+        internal val TAG: String = VGSCollect::class.qualifiedName.toString()
```

Comment:
@Endorf I would suggest moving this constant to VGSCollectLogger and make it default.